recipes/meta.yaml

```diff
@@ -45,7 +45,7 @@ requirements:
     - geopandas>=0.6.1
     - matplotlib>=2.1.0
     - networkx>=2.2
-    - pandas>=0.24.1,<2
+    - pandas>=0.24.1
```

Comment:
now sure how to test but does it still build with pandas that is < 2?

Reply:
So should I test it as is? by running a new install of this pyincore branch? will it update the pandas version?

Reply:
To test this, I think you can do two things:
1) Create a conda environment for the 1.10.0 release (which has pandas < 2) to verify it still works with less than version 
2) Then you could update the conda environment to pandas 2 and verify it still works.

Reply:
Nevermind, you'd need to copy the new analyses over. If you can, you could use a pip virtualenv so you can install the updated analyses and test before and after upgrading pandas.

Reply:
I verified that pandas>= 1.1.0 works and updated the recipes of publish accordingly.

Reply:
any reason not to make that the minimum version? That was released July 1, 2020

Reply:
Lower than that won't work anymore @robkooper 
```
 pandas 1.0.0 - doesn't work anymore
AttributeError: module 'numpy' has no attribute 'bool'.
np.bool was a deprecated alias for the builtin bool. To avoid this error in existing code, use bool by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use np.bool_ here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
```